modules/appeals_api/spec/fixtures/decision_reviews/v2/valid_200995_extra.json

```diff
@@ -16,9 +16,11 @@
         "OTHER"
       ],
       "homelessLivingSituationOther": "random words for other living situation ",
-      "homelessPointOfContact": "Jim Jones Vet Care Network",
+      "homelessPointOfContact": "Shawn Hobkens Veteran Care Network",
```

Comment:
![a man in a suit and tie making a funny face](https://media.tenor.com/zqHbKTg9NxEAAAAC/ray-liotta-laughing.gif)